.github/workflows/SnoopCompile.yml

```
@@ -0,0 +1,97 @@
+name: SnoopCompile
+
+on:
+  push:
+    branches:
+    #  - 'master'  # NOTE: to run the bot only on pushes to master
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  SnoopCompile:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    env:
+      GKS_ENCODING: "utf8"
+      GKSwstype: "100"
+      PLOTS_TEST: "true"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:   # NOTE: the versions below should match those in your botconfig
+          - '1.3'
+          - '1.4'
+          # - '1.5'
```

Enabling this since nightly is 1.6 now? 🤔 
```suggestion
           - '1.5'
```

---

I tried but it did not work: https://github.com/daschw/Plots.jl/actions/runs/153176644

---

All 1.5 tests are failing there.

---

Maybe try `-'1.5.0-rc1'`.

@SaschaMann shouldn't 1.5 install `'1.5.0-rc1'` by default? julia-setup fails to do this.

---

ok, let's try

---

> @SaschaMann shouldn't 1.5 install '1.5.0-rc1' by default? julia-setup fails to do this.

`1.5` doesn't include prereleases, you need to use `1.5.0-0` to include all prereleases.

(I'm in the process of adding 1.5.0-rc1 right now, right now it points at 1.5.0-beta1)

---

So, `1.5.0-0` would be recommended?

---

Yes.

(1.5.0-rc1 should be available now)

---

Thanks!

---

With `1.5.0-0` I get
```

Run julia-actions/setup-julia@latest
0s
    PLOTS_TEST: true
Run julia-actions/setup-julia@latest
  with:
    version: 1.5.0-0
    arch: x64
    show-versioninfo: false
  env:
    GKS_ENCODING: utf8
    GKSwstype: 100
    PLOTS_TEST: true
##[error]Unexpected HTTP response: 404
```
https://github.com/daschw/Plots.jl/runs/824362165

---

My bad, it should've been `^1.5.0-0`. Sorry! 🤦

---

That does not work either: https://github.com/daschw/Plots.jl/actions/runs/155304982

---

But that's an issue with SnoopCompile.jl cc: @aminya